### PR TITLE
CI(actions): Add autopep8 to pre-commit hooks

### PR DESCRIPTION
**Pull request by mtache**
## Change Summary

Add autopep8 to pre-commit hooks and configure with .pep8 file as required by Ansible linter.
Do a run to sanitize the repo.

## Component(s) name

pre-commit

## Proposed changes
Added hook using https://github.com/pre-commit/mirrors-autopep8
Configure via .pep8 as per Ansible linter requirements when running `make sanity`
Propose new Makefile targets

## How to test
pre-commit run autopep8 --all-files

### User Checklist

Done

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly. (check the box if not applicable)
- [x] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -9,6 +9,10 @@ repos:
       - id: check-yaml
       - id: check-added-large-files
       - id: check-merge-conflict
+  - repo: https://github.com/pre-commit/mirrors-autopep8
+    rev: 898691a48e7dbdbbac32f44f9336ef944d74cf82
```
Comment (gmuloc) [inline .pre-commit-config.yaml]: Any tag maybe to use here?

File: ansible_collections/arista/cvp/plugins/modules/cv_change_control_v3.py [hunk 2]
```diff
@@ -19,6 +19,10 @@
 #
 
 from __future__ import absolute_import, division, print_function
+from ansible_collections.arista.cvp.plugins.module_utils.change_tools import CvChangeControlTools
+from ansible_collections.arista.cvp.plugins.module_utils import tools_cv
+from ansible.module_utils.basic import AnsibleModule
+import logging
```
Comment (gmuloc) [inline ansible_collections/arista/cvp/plugins/modules/cv_change_control_v3.py]: The comment below still make sense?

File: ansible_collections/arista/cvp/plugins/modules/cv_configlet.py [hunk 3]
```diff
@@ -19,6 +19,12 @@
 #
 
 from __future__ import absolute_import, division, print_function
+from ansible_collections.arista.cvp.plugins.module_utils import schema_v1 as schema
```
Comment (gmuloc) [inline ansible_collections/arista/cvp/plugins/modules/cv_configlet.py]: Somehow thought there was also a rule about order of imports if they were built in vS non built in but I guess auto pep8 is happy this way?

File: .pre-commit-config.yaml (hunk 1 above)
Comment (mtache) [reply .pre-commit-config.yaml]: nope, c.f. README on the hook repo. Need to use a SHA unfortunately

File: ansible_collections/arista/cvp/plugins/modules/cv_configlet.py (hunk 3 above)
Comment (mtache) [reply ansible_collections/arista/cvp/plugins/modules/cv_configlet.py]: First configuration attempt was wrong. We need to ignore E402 - Fix module level import not at top of file in autopep8 to make ansible-test happy. Ansible requires imports to be after the documentation string...
Fixed in another commit. Imports stays were they are

File: ansible_collections/arista/cvp/plugins/modules/cv_change_control_v3.py (hunk 2 above)
Comment (mtache) [reply ansible_collections/arista/cvp/plugins/modules/cv_change_control_v3.py]: It does

File: .pre-commit-config.yaml (hunk 1 above)
Comment (mtache) [reply .pre-commit-config.yaml]: Misread the README, you are right. Using v1.6.0 tag

**Review by gmuloc — APPROVED** [review]
LGTM

**Comment by gmuloc** [comment]
> LGTM

ignore this - CI is failing .. time for coffee

**Pull request closed without merging**